packages/admin-next/dashboard/src/routes/product-variants/product-variant-detail/components/variant-prices-section/variant-prices-section.tsx

```diff
@@ -0,0 +1,82 @@
+import { useTranslation } from "react-i18next"
+import { useState } from "react"
+
+import { CurrencyDollar } from "@medusajs/icons"
+import { Button, Container, Heading } from "@medusajs/ui"
+import { MoneyAmountDTO, ProductVariantDTO } from "@medusajs/types"
+
+import { ActionMenu } from "../../../../../components/common/action-menu"
+import { getLocaleAmount } from "../../../../../lib/money-amount-helpers"
+import { NoRecords } from "../../../../../components/common/empty-table-content"
+
+type VariantPricesSectionProps = {
+  variant: ProductVariantDTO & { prices: MoneyAmountDTO[] }
+}
+
+export function VariantPricesSection({ variant }: VariantPricesSectionProps) {
+  const { t } = useTranslation()
+
+  const prices = variant.prices
+    .filter((p) => !p.rules?.length)
+    .sort((p1, p2) => p1.currency_code?.localeCompare(p2.currency_code)) // display just currency prices
+
+  const [current, setCurrent] = useState(Math.min(prices.length, 3))
+
+  const hasPrices = !!prices.length
+
+  const displayPrices = prices.slice(0, current)
+
+  const onShowMore = () => {
+    setCurrent(Math.min(current + 3, prices.length))
+  }
+
+  return (
+    <Container className="flex flex-col divide-y p-0">
+      <div className="flex items-center justify-between px-6 py-4">
+        <Heading level="h2">{t("labels.prices")}</Heading>
+        <ActionMenu
+          groups={[
+            {
+              actions: [
+                {
+                  label: t("actions.edit"),
+                  to: `/products/${variant.product_id}/prices`,
```

Comment:
q: shouldn't this open a bulk editor with only this variants prices? If we go to the "big" editor the user might have to scroll to the bottom find the prices they want to edit.

Reply:
+1 if possible!

Reply:
@olivermrbl could you check the flows with the price editor now

Reply:
Love this!

Minor, minor thing: Should we add breadcrumbs to indicate where in the product details page stack you are?

![CleanShot 2024-06-19 at 13 19 54](https://github.com/medusajs/medusa/assets/59018053/53c5e534-96d1-4f68-8afe-bb301b818399)

Reply:
For that we would need to restructure product components and routing a bit and I wouldn't block this PR with this if possible.